### PR TITLE
swift2 compatible, added guard statements to ciImage, fixed failing tests, updated .travis.yml

### PR DESCRIPTION
**Pull request by schultzjim**
Having trouble getting Travis to pass even after updating `osx_image: xcode6.4` to `osx_image: xcode7`

### CODE REVIEW

**Comment by schultzjim** [comment]
keeps choking here 

```
2015-09-18 18:12:02.661 xctool[599:2773] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'XCTestConfiguration isn't available'
*** First throw call stack:
(
    0   CoreFoundation                      0x00007fff86bca03c __exceptionPreprocess + 172
    1   libobjc.A.dylib                     0x00007fff9028476e objc_exception_throw + 43
    2   CoreFoundation                      0x00007fff86bc9e1a +[NSException raise:format:arguments:] + 106
    3   Foundation                          0x00007fff8caa599b -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 195
    4   xctool                              0x00000001039bd7ff -[OCUnitTestRunner testEnvironmentWithSpecifiedTestConfiguration] + 204
    5   xctool                              0x00000001039bacd6 -[OCUnitIOSAppTestRunner runTestsAndFeedOutputTo:startupError:otherErrors:] + 1654
    6   xctool                              0x00000001039bc92d -[OCUnitTestRunner runTests] + 195
    7   xctool                              0x00000001039ca3de __158-[RunTestsAction blockForTestable:focusedTestCases:allTestCases:testableExecutionInfo:testableTarget:isApplicationTest:arguments:environment:testRunnerClass:]_block_invoke + 311
    8   xctool                              0x00000001039cc131 __56-[RunTestsAction runTestables:options:xcodeSubjectInfo:]_block_invoke404 + 336
    9   xctool                              0x00000001039cb867 -[RunTestsAction runTestables:options:xcodeSubjectInfo:] + 4998
    10  xctool                              0x00000001039c98b2 -[RunTestsAction performActionWithOptions:xcodeSubjectInfo:] + 1234
    11  xctool                              0x00000001039de0d3 -[TestAction performActionWithOptions:xcodeSubjectInfo:] + 106
    12  xctool                              0x00000001039b534d -[XCTool run] + 3334
    13  xctool                              0x00000001039b3c7c main + 1064
    14  xctool                              0x00000001039b0c24 start + 52
)
libc++abi.dylib: terminating with uncaught exception of type NSException
/Users/travis/build.sh: line 41:   599 Abort trap: 6           xctool -scheme QRCodeTests -sdk iphonesimulator build test
The command "xctool -scheme QRCodeTests -sdk iphonesimulator build test" exited with 134.
Done. Your build exited with 1.
```

**Comment by aschuch** [comment]
I am not really a fan of returning an empty CIImage in case of a failure. Maybe we should return nil in case of an error?
I know that the filter might fail since they are so loosely typed and instantiated using string constants, but I can't come up with a case that would actually crash this part of the code base.

**Comment by schultzjim** [comment]
makes sense to me, i'll make that change

**Comment by schultzjim** [comment]
k got those changes in place, let me know what you think or if there anything else i can adjust etc.

File: Example/ViewController.swift [hunk 1]
```diff
@@ -22,7 +22,7 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        CGSize(width: 300, height: 300)
+//        CGSize(width: 300, height: 300)
```
Comment (aschuch) [inline Example/ViewController.swift]: I think we could get rid if this entirely.

File: QRCode/QRCode.swift [hunk 2]
```diff
@@ -63,30 +63,32 @@ public struct QRCode {
...
-
+    
     // MARK: Generate QRCode
     
     /// The QRCode's UIImage representation
-    public var image: UIImage {
-        return UIImage(CIImage: ciImage)!
+    public var image: UIImage? {
+        return UIImage(CIImage: ciImage!)
     }
```
Comment (aschuch) [inline QRCode/QRCode.swift:79]: ciImage should not be forced unwrapped here, we should probably guard against that here as well.

File: Example/ViewController.swift (hunk 1 above)
Comment (schultzjim) [inline Example/ViewController.swift]: ha ha ya absolutely, i'll do that now

**Comment by aschuch** [comment]
Thanks for the changes :heart:

**Comment by schultzjim** [comment]
no prob, this is a great project, thanks for building it.

File: QRCode/QRCode.swift (hunk 2 above)
Comment (schultzjim) [inline QRCode/QRCode.swift:79]: ya thats perfect, overlooked that one. committing the change now

**Comment by aschuch** [comment]
Thanks :boom:

**Pull request merged**